### ISSUE TITLE
oAuth 1

### ISSUE THREAD

**Issue by roykho** [rangav/thunder-client-support#176]
Would be nice to have support for oAuth 1.0 like in Postman!

Thanks!

**Comment by rangav** [comment]
Hi @roykho will add to roadmap.

**Comment by deniscass** [comment]
Hi Guys, will be great add oAuth 1. I'm a Magento 1 developer and i need this auth to test API/REST from Magento

**Comment by rangav** [comment]
@sheppe Jira seems to have OAuth 2.0
https://developer.atlassian.com/cloud/jira/platform/oauth-2-3lo-apps/

**Comment by rangav** [comment]
Outh2 that's how they work, GitHub twitter api they all use similar model.

You can use Oauth2 for Rest api calls, please try and see.

they also have basic auth
https://developer.atlassian.com/cloud/jira/platform/basic-auth-for-rest-apis/

**Comment by sheppe** [comment]
> Outh2 that's how they work, GitHub twitter api they all use similar model.
> 
> You can use Oauth2 for Rest api calls, please try and see.
> 
> they also have basic auth https://developer.atlassian.com/cloud/jira/platform/basic-auth-for-rest-apis/

Yup, I understand that, but it's still not an option for all use cases. We have an enterprise edition we're running and oauth1 is the only option I'm permitted to use.

Also, for your reference: https://developer.atlassian.com/server/jira/platform/rest-apis/#authentication-and-authorization

**Comment by rangav** [comment]
Ok got it
